pandas/formats/format.py

```diff
@@ -2260,6 +2260,35 @@ def _format_strings(self):
         return fmt_values
 
 
+def format_percentiles(percentiles):
+    '''
```

Comment:
I think that you can put the raise on non-unique check in this routine, its one of your guarantees